src/skin/options-layout.css

```diff
@@ -82,30 +88,38 @@ button
 }
 
 .switch-container{
-  float: right;
   margin-left: 0;
-  margin-right: 23px;
-
+  position: absolute;
+  display: inline;
 }
 
 .options .switch-container {
   max-width: none;
 }
 
+.switch-toggle {
+  width: 115px;
+}
+
 .honeybadgerPowered{
-  float: right;
-  left: 148px;
   bottom: 0;
+  display: inline-block !important;
```

Comment:
Anything worth documenting here? Why the "!important"?

Reply:
Changed this to use a more specific selector to avoid use of '!important' - it was only used to overwrite something set in `popup.css`.